plugin-test/src/test/groovy/org/gradle/github/dependency/extractor/MultiProjectDependencyExtractorTest.groovy

```diff
@@ -72,38 +80,30 @@ class MultiProjectDependencyExtractorTest extends BaseExtractorTest {
 
     def "extracts transitive project dependencies in multi-project build"() {
         given:
-        multiProjectBuild("parent", ["a", "b", "c"]) {
-            List<BuildTestFile> projects = []
-            projects.add project("a").tap {
-                buildFile """
+        settingsFile << "include 'a', 'b', 'c'"
+        buildFile << """
+            project(':a') {
```

Comment:
IIRC, the idiomatic gradle project was aiming to drop support for this syntax within Gradle for a couple of reasons, one of which was that it doesn't work with `configureOnDemand` and some other reasons I can't remember right now.

The replacement was supposed to be buildscript plugins to centralize logic.

Reply:
This is a convenient mechanism to declare a multi-project build in a test, and results in the same dependencies being resolved as other approaches. Until this mechanism is actually deprecated, I'd prefer to keep things simple.